pkg/faketikv/cases/add_nodes.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cases
+
+import (
+	"github.com/pingcap/kvproto/pkg/metapb"
+	"github.com/pingcap/pd/server/core"
+	log "github.com/sirupsen/logrus"
+)
+
+func newAddNodes() *Conf {
+	var conf Conf
+
+	for i := 1; i <= 3; i++ {
+		conf.Stores = append(conf.Stores, Store{
```

Comment:
seems that it can be extracted to a function, almost every case need this.

Reply:
The details could be different case by case.